.github/workflows/license_tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Install System Dependencies
         run: |
           sudo apt-get update
-          sudo apt install python3-dev swig libssl-dev
+          sudo apt install python3-dev swig libssl-dev portaudio19-dev
```

Comment:
since you are touching this file, can you fix the tests and use the latest workflow?